nuxeo-services/nuxeo-platform-directory/nuxeo-platform-directory-ldap/src/test/resources/log4j2-test.xml

```diff
@@ -12,6 +12,9 @@
   </Appenders>
   <Loggers>
     <Logger name="org.nuxeo" level="info" />
+    <!-- Set level to ERROR to ignore warning explaining that admin password needs to be changed,
+         as it cannot be changed before startup. -->
+    <Logger name="org.apache.directory.server.core.DefaultDirectoryService" level="error"/>
```

Comment:
Shouldn't we propagate this to all `log4j2-test.xml` ?

Reply:
Since it is only linked to LDAP tests when starting a mock server, isn't it enough ? If not, will reopen it.